core/disputes.go

```diff
@@ -308,6 +308,9 @@ func (n *OpenBazaarNode) ProcessDisputeOpen(rc *pb.RicardianContract, peerID str
 		// Load out version of the contract from the db
 		myContract, state, _, records, _, _, err := n.Datastore.Sales().GetByOrderId(orderID)
 		if err != nil {
+			if err := n.SendProcessingError(DisputerID, orderID, pb.Message_DISPUTE_OPEN, nil); err != nil {
+				log.Errorf("failed sending ORDER_PROCESSING_FAILURE to peer (%s): %s", DisputerID, err)
+			}
```

Comment:
If a order is not present in the node db, does it mean there is a processing error.
A typical scenario where the seller was offline and there are 2 offline messages awaiting processing - ORDER and ORDER_PAYMENT and ORDER_PAYMENT was processed first, then it not really any order processing error. 
It would be better if the processing error was flagged and checked after all the offline messages were processed before send a processing error to peer.

Reply:
In the offline case, all available offline messages are downloaded in bulk and processed in a deterministic order (to ensure that this failure is, in fact, due to the message being unavailable).